src/parser/parser.yy

```diff
@@ -537,6 +537,9 @@ create_schema_prop_list
 
  create_schema_prop_item
     : KW_TTL_DURATION ASSIGN INTEGER {
+        if ($3 < 0) {
```

Comment:
what's your desgin for ttl == 0?

Reply:
< = 0 represents infinity.
so `<0` to `=0`
Here I add a comment.